components/compliance-service/reporting/relaxting/util_test.go

```diff
@@ -1,23 +1,30 @@
 package relaxting
 
 import (
-	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
+
+	"github.com/chef/automate/components/compliance-service/ingest/ingestic/mappings"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestGetEsIndexNoTimeLast24h(t *testing.T) {
 	sum := mappings.IndexNameSum
 	filters := make(map[string][]string)
 
-	index, err := GetEsIndex(filters, true)
-
-	today := time.Now().UTC().Format("2006.01.02")
-	yesterday := time.Now().Add(-24 * time.Hour).UTC().Format("2006.01.02")
+	todayTime := time.Now().UTC()
 
-	assert.Equal(t, sum+"-"+yesterday+"*,"+sum+"-"+today+"*", index)
+	index, err := GetEsIndex(filters, true)
 	assert.NoError(t, err)
+
+	today := todayTime.Format("2006.01.02")
+	if today[len(today)-2:] == "31" {
```

Comment:
These can be written using the `Day` function:
`if todayTime.Day() == 31 `

Reply:
sorry @vivek-yadav.. you're taking the heat for the change that I made and you were nice enough to create a PR for.. ;).. @kalroy yes, i agree, I could have used the day func but just got caught up in the slice approach